thirdeye/thirdeye-frontend/app/pods/components/rootcause-settings/template.hbs

```diff
@@ -13,16 +13,15 @@
           </label>
           {{date-range-picker
             class="te-range-picker"
-            timePicker=showTimePicker
+            timePicker=true
             timePicker24Hour=true
-            timePickerIncrement=timePickerIncrement
-            minDate=minDate
-            maxDate=maxDate
+            timePickerIncrement=5
             start=datePickerAnomalyRangeStart
             end=datePickerAnomalyRangeEnd
+            maxDate=datePickerMaxDate
             ranges=anomalyRanges
             showCustomRangeLabel=false
-            format=uiDateFormat
+            format="MMM D, YYYY hh:mm a"
```

Comment:
fyi the `timePicker`  `uiDateFormat` here was an ask from Kishore, he wanted the date ranges to depend on the granularity

Reply:
yes. if it ever comes up, please blame it on me. :-)